README.md

```diff
@@ -97,7 +97,7 @@ You can run the tests with the commands below:
 ```
 cd manticore
 # all tests
-nosetests
+nosetests --exe
```

Comment:
Sure thing, I'll take it out. I tried cloning the repo again, and I still have the tests being executable; it's probably an OS-specific thing. Do you think it makes sense to add a note saying maybe try the flag if the tests don't seem to run, or make sure that the tests are set to non executable?